docs/phylum.md

```diff
@@ -12,6 +12,7 @@ Interact with the Phylum API
 * [phylum project](https://docs.phylum.io/docs/phylum_project)
 
 ### Additional Commands
+* [phylum group](https://docs.phylum.io/docs/phylum_group)
```

Comment:
Where is this file?

Reply:
https://github.com/phylum-dev/cli/blob/main/docs/phylum_group.md

It is new and won't be live until we release.

Reply:
Oh... They already existed, but weren't linked. My bad.